### DISCUSSION TITLE
question? evil-org-mode version

### DISCUSSION THREAD

**Discussion by jgarte** [syl20bnr/spacemacs#15478] [Q&A] [answered]
Are we still using edwtjo instead of the [maintained repo](https://github.com/Somelauw/evil-org-mode)?

https://www.spacemacs.org/layers/+emacs/org/README.html

https://github.com/edwtjo/evil-org-mode

**Comment by dankessler** [comment, marked as answer]
Running `describe-package` (`C-h P evil-org RET`)

shows

```
Package evil-org is installed.

     Status: Installed in ‘evil-org-20220227.1024/’ (unsigned). Delete
    Version: 20220227.1024
     Commit: 0d10ff7bb9a3a93d25cd91018b17f0a052b335f3
    Summary: evil keybindings for org-mode
   Requires: emacs-24.4, evil-1.0
    Website: https://github.com/Somelauw/evil-org-mode.git
   Keywords: evil vim-emulation org-mode key-bindings presets 
 Maintainer: Somelauw
Other versions: 20220227.1024 (melpa).


Known Bugs:
See, https://github.com/Somelauw/evil-org-mode/issues

[back]
```

which suggests to me that this is just being installed from MELPA, and [MELPA's recipe](https://github.com/melpa/melpa/blob/master/recipes/evil-org) (appropriately) points to the maintained repo you linked.
